sdk/keyvault/azure-keyvault-keys/src/samples/java/com/azure/security/keyvault/keys/KeyClientJavaDocCodeSnippets.java

```diff
@@ -352,6 +352,18 @@ public void listKeySnippets() {
                 keyWithMaterial.keyMaterial().kty());
         }
         // END: com.azure.keyvault.keys.keyclient.listKeys#Context
+
+        // BEGIN: com.azure.keyvault.keys.keyclient.listKeys.iterableByPage
+        keyClient.listKeys().iterableByPage().forEach(resp -> {
+            System.out.printf("Response headers are %s. Url %s  and status code %d %n", resp.headers(),
```

Comment:
Is this going to `toString` correctly for the headers?

If you want to do associations for URL and status code show it like this:

`URL: %s, Status code: %d`